src/styles/blocks/header.scss

```diff
@@ -0,0 +1,20 @@
+.header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  @include font;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding: 0 50px;
+
+  &__logo {
+    display: flex;
+    padding-right: 243px;
+
+    @include screen-xl {
+      padding-right: 419px;
```

Comment:
The padding-right value of `419px` inside the `screen-xl` mixin might also cause layout issues on different screen sizes. Ensure that this value is appropriate for the intended screen size and consider using relative units if possible.